src/EnergyPlus/SystemAvailabilityManager.cc

```diff
@@ -792,7 +792,7 @@ namespace SystemAvailabilityManager {
 				SetupOutputVariable( "Availability Manager Optimum Start Control Status []", OptStartSysAvailMgrData( SysAvailNum ).AvailStatus, "System", "Average", OptStartSysAvailMgrData( SysAvailNum ).Name );
 
 				// add
-				SetupOutputVariable( "Availability Manager Optimum Start Hours Before Occupancy []", OptStartSysAvailMgrData( SysAvailNum ).NumHoursBeforeOccupancy, "System", "Average", OptStartSysAvailMgrData( SysAvailNum ).Name, "Daily" );
```

Comment:
Hmmmm, @kbenne, what are your thoughts on this one.  Also @macumber.  This adds the units to the output variable.  Which presumably changes the output schema?  I wouldn't mind merging these changes in for the bug fix release @mjwitte except for this potential issue.